tests/UnitTests/SharedTests/MappingUtilitiesTests/MappingUtilitiesTests.cs

```diff
@@ -0,0 +1,116 @@
+using NHS.CohortManager.Shared.Utilities;
+
+namespace MappingUtilitiesTests;

```

Comment:
```suggestion
namespace NHS.CohortManager.Tests.Shared;
```
and move the namespace to the first line in the file pls